canned/rabbitmq.json

```diff
@@ -0,0 +1,72 @@
+{
+	"id": "0c57a644-aa74-4ec3-b099-b44499df1159",
+	"measurement": "rabbitmq_node",
+	"app": "rabbitmq",
+			"cells": [
+				{
+					"x": 0,
+					"y": 0,
+					"w": 4,
+					"h": 4,
+					"i": "c46351c6-b33a-4dc2-a053-3517e7c8098e",
+					"name": "RabbitMQ - Overview",
+					"queries": [
+						{
+							"query": "select mean(\"consumers\") AS \"consumers\" from rabbitmq_overview",
+							"groupbys": [],
+							"wheres": []
+						},
+						{
+							"query": "select mean(\"exchanges\") AS \"exchanges\" from rabbitmq_overview",
+							"groupbys": [],
+							"wheres": []
+						},
+						{
+							"query": "select mean(\"queues\") AS \"queues\" from rabbitmq_overview",
+							"groupbys": [],
+							"wheres": []
+						}
+					]
+				},
+				{
+					"x": 0,
+					"y": 0,
+					"w": 4,
+					"h": 4,
+					"i": "c46351c6-b33a-4dc2-a053-3517e6c8098e",
```

Comment:
Sadly, all these "i" fields need to be unique ids.  I have a script in the canned directory called uuid.sh.  Run that to get a few of 'em.

I'd like to see "i" removed in the future, but, for now you'll need to ad that.